### ISSUE TITLE
2.0.1 release

### ISSUE THREAD

**Issue by adamralph** [FakeItEasy/FakeItEasy#733]
**Ready** when all other issues on this milestone are **Done** and closed.

- [ ] run code analysis in VS in *Release* mode and address violations (send a regular PR which must be merged before continuing)
- [ ] if necessary, change `VERSION_SUFFIX` on [CI Server](http://teamcity.codebetter.com/admin/editBuildParams.html?id=buildType:bt929)
      to appropriate "-beta123" or "" (for non-betas) value and initiate a build
- [ ] check build
-  edit draft release in [GitHub UI](https://github.com/FakeItEasy/FakeItEasy/releases):
    - [ ] complete release notes, mentioning non-owner contributors, if any (move release notes forward from any pre-releases to the current release)
    - [ ] attach nupkg
    - [ ] publish the release
- [ ] push NuGet package
- [ ] de-list pre-release or superseded buggy NuGet packages if present
- [ ] update website with contributors list (if in place)
- [ ] tweet, mentioning contributors and post link as comment here for easy retweeting ;-)
- [ ] post tweet in [Gitter](https://gitter.im/FakeItEasy/FakeItEasy)
- [ ] post links to the NuGet and GitHub release in each issue in this milestone, with thanks to contributors
- [ ] use `rake set_version[new_version]` to
    - create a new branch
    - change CommonAssemblyInfo.cs to expected minor version (of form _xx.yy.zz_)
    - push to origin
    - create PR to upstream master
- [ ] use `rake create_milestone` to
    - create a new milestone for the next release
    - create new issue (like this one) for the next release, adding it to the new milestone
    - create a new draft GitHub Release
- [ ] close this milestone

**Comment by adamralph** [comment]
Not required - we will go straight to 2.1.0 since #736 is now merged.
